components/Common/ForumSettingsBase.cs

```diff
@@ -73,14 +73,13 @@ public string Theme
 		{
 			get
 			{
-                return Settings.GetString(SettingKeys.Theme, "_default");
+                return Settings.GetString(SettingKeys.Theme, "community");
```

Comment:
Not a breaking change as it's the default when installing a new forums instance. I just like "community" better than "_default". And since default is retained during an upgrade, an administrator could just swap to community if desired.

Reply:
Oh, so is this a new theme or an existing theme?  I'm still missing the context.  Sorry.  :(

Reply:
"community" is the theme from dnncommunity.org. The old one is "_default" and it's still there. The new one is "community". So when a new install is done it uses "community" as the theme when setting up the forums.